src/ale_interface.cpp

```diff
@@ -217,6 +218,17 @@ void ALEInterface::loadROM(std::string rom_file) {
 #endif
 }
 
+bool ALEInterface::isSupportedRom(){
```

Comment:
I don't understand the point of this, you call `isSupportedROM` after you've already loaded a ROM and presumably been warned if it was or wasn't supported? If this is going to be a public API shouldn't it take a filename and actually check the MD5 against the filename. I'm assuming you want an API like this for your ROM loading package?

Reply:
This makes a lot more sense. I'll make that change.

Reply:
Made this a static method so it can be called without loading a ROM